Index.php changes

fbane opened the pull request:
- Cleaned up folders names to ensure they get created
- Reset path for $db variable
- Reduced data for main table and condensed some data
- Updated variable for time check
- Link created now for only files in saves/public/ folder

fbane committed:
Updated for proper upload link

fbane committed:
Updates to index.php

- Reduced table headers to fit better
- Removed and updated variable for time check

fbane committed:
Update to index.php

Reset folder names to be created if not found

ian76g commented on index.php:
```diff
@@ -25,8 +25,7 @@
-                        <th><A href="?sortby=2&sortorder=desc" style="color: white">Locos</A></th>
-                        <th><A href="?sortby=3&sortorder=desc" style="color: white">Carts</A></th>
+                        <th><A href="?sortby=6&sortorder=desc" style="color: white">Rolling Stock</th>
                         <th><A href="?sortby=4&sortorder=desc" style="color: white">Slope</A></th>
```
sortby=6 means sort by trees.
you have to be able to sort by locos and carts
As I had trees and switches in one column, you asked me to have each value in its own column

ian76g commented on index.php:
```diff
@@ -120,29 +121,33 @@ function mysort($a, $b)
+
+                              // Timecheck to remove public link for download
+                                if ($saveFile < $timeCheck) {
+                                    echo '<td><a href="'.WWW_ROOT.'saves/public/'.$saveFile.'">Link</a></td>';
```
saveFile is a string, timeCheck is a number.
This comparison makes no sense here

fbane committed:
Index changes

- Rewrote table headers
- Added time check to remove DL link for public saves
- Corrected column sorted
